app/screens/send-bitcoin-screen/send-bitcoin-completed-screen.tsx

```diff
@@ -198,6 +213,70 @@ const useStyles = makeStyles(({ colors }) => ({
     padding: 20,
     borderRadius: 20,
   },
+  truncatedText: {
```

Comment:
seems the styles in this files are duplicated in other componentes... please remove them if they dont belong to this file